discovery-provider/alembic/versions/8c10373f2615_user_tips.py

```diff
@@ -0,0 +1,76 @@
+"""User Tips
+
+Revision ID: 8c10373f2615
+Revises: 2cfd17a60647
+Create Date: 2022-04-14 00:06:55.572866
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "8c10373f2615"
+down_revision = "2cfd17a60647"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # USER TIPS
+    op.create_table(
```

Comment:
I think this needs to be idempotent per all new migrations - so like create is not exists